main/src/cgeo/geocaching/CacheListActivity.java

```diff
@@ -1536,9 +1536,9 @@ public void onClick(final View arg0) {
             showProgress(true);
             showFooterLoadingCaches();
 
-            getSupportLoaderManager().restartLoader(CacheListLoaderType.NEXT_PAGE.getLoaderId(), null, CacheListActivity.this);
+            LoaderManager.getInstance(CacheListActivity.this).restartLoader(CacheListLoaderType.NEXT_PAGE.getLoaderId(), null, CacheListActivity.this);
```

Comment:
Why not use `LoaderManager.getInstance(this).` as with the other changes?

Reply:
The instance of the type variable has to exist, so `this` has to be explicitly referred to as the activity.